charts/gardener-extension-admission-kubevirt/charts/application/templates/validatingwebhook-validator.yaml

```diff
@@ -29,3 +30,27 @@ webhooks:
       path: /webhooks/validate
     {{- end }}
     caBundle: {{ required ".Values.webhookConfig.caBundle is required" (b64enc .Values.global.webhookConfig.caBundle) }}
+- name: secrets.validation.gcp.provider.extensions.gardener.cloud
```

Comment:
I think that should be `secrets.validation.kubevirt.provider.extensions.gardener.cloud`

Reply:
Thanks for noticing, fixed.